Tests/CryptomatorCloudAccessTests/WebDAV/PropfindResponseCacheTests.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  PropfindResponseCacheTests.swift
+//  CryptomatorCloudAccessTests
+//
+//  Created by Philipp Schmid on 07.07.22.
+//  Copyright © 2022 Skymatic GmbH. All rights reserved.
+//
+
+import GRDB
+import XCTest
+#if canImport(CryptomatorCloudAccessCore)
+@testable import CryptomatorCloudAccessCore
+#else
+@testable import CryptomatorCloudAccess
+#endif
+/*
```

Comment:
What about these tests?

Reply:
Ah thanks this were some early unit tests before we unified the two implementations as `DirectoryContentCache`.
Resolved by 5206564e96492d4c428fdc207e2f12e8f1fdb964.